services/cleanup/regular.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from shared.django_apps.reports.models import ReportDetails
+
+from services.cleanup.cleanup import run_cleanup
+from services.cleanup.utils import CleanupResult, CleanupSummary
+
+log = logging.getLogger(__name__)
+
+
+def run_regular_cleanup() -> CleanupSummary:
```

Comment:
Shouldn't we have some sort of locking or deduplication here to prevent multiple processes trying to delete the same things?

I'm not sure if `run_cleanup` already does this, or if we are delegating to the DB to return different results when we query...
Maybe it's fine to have multiple processes trying to delete the same thing, just confirming.

Reply:
I can think of one case in which it would be bad to have two concurrent processes running the same query, which I can trivially fix. Otherwise this should be fine to run concurrently.